jaeger-core/src/main/java/com/uber/jaeger/senders/ThriftSender.java

```diff
@@ -17,67 +17,52 @@
 import com.uber.jaeger.Span;
 import com.uber.jaeger.exceptions.SenderException;
 import com.uber.jaeger.reporters.protocols.JaegerThriftSpanConverter;
-import com.uber.jaeger.reporters.protocols.ThriftUdpTransport;
+import com.uber.jaeger.thrift.reporters.protocols.ThriftUdpTransport;
+import com.uber.jaeger.thrift.senders.ThriftSenderBase;
```

Comment:
I am not sure I fully understand this change. Don't these imports still retain the dependency on jaeger-thrift?